apis/management.cattle.io/v3/schema/schema.go

```diff
@@ -499,8 +499,44 @@ func logTypes(schema *types.Schemas) *types.Schemas {
 			m.DisplayName{}).
 		AddMapperForType(&Version, v3.ProjectLogging{},
 			m.DisplayName{}).
-		MustImport(&Version, v3.ClusterLogging{}).
-		MustImport(&Version, v3.ProjectLogging{})
+		MustImport(&Version, v3.ClusterTestInput{}).
+		MustImport(&Version, v3.ProjectTestInput{}).
+		MustImportAndCustomize(&Version, v3.ClusterLogging{}, func(schema *types.Schema) {
+			schema.ResourceActions = map[string]types.Action{
+				"test": {
+					Input: "clusterTestInput",
+				},
+				"dryRun": {
+					Input: "clusterTestInput",
+				},
+			}
+			schema.CollectionActions = map[string]types.Action{
+				"test": {
+					Input: "clusterTestInput",
+				},
+				"dryRun": {
+					Input: "clusterTestInput",
+				},
+			}
+		}).
+		MustImportAndCustomize(&Version, v3.ProjectLogging{}, func(schema *types.Schema) {
+			schema.ResourceActions = map[string]types.Action{
```

Comment:
If we only support collection action, we should remove the resource action here.